content/en/getting-started/quickstart/index.md

```diff
@@ -0,0 +1,314 @@
+{{< /alert >}}
+
+#### Create the S3 buckets
+
+{{< command >}}
+$ awslocal s3 mb s3://localstack-thumbnails-app-images
+$ awslocal s3 mb s3://localstack-thumbnails-app-resized
+{{< / command >}}
+
+#### Add bucket names into the parameter store
+
+{{< command >}}
+$ awslocal ssm put-parameter \
+    --name /localstack-thumbnail-app/buckets/images \
+    --type "String" \
+    --value "localstack-thumbnails-app-images"
+$ awslocal ssm put-parameter \
+    --name /localstack-thumbnail-app/buckets/resized \
+    --type "String" \
+    --value "localstack-thumbnails-app-resized"
+{{< / command >}}
+
+#### Create SNS DLQ Topic for failed lambda invocations
+
+{{< command >}}
+$ awslocal sns create-topic --name failed-resize-topic
+{{< / command >}}
+
+To receive immediate alerts in case of image resize failures, subscribe an email address to the system. You can use the following command to subscribe an email address to the SNS topic:
+
+{{< command >}}
+$ awslocal sns subscribe \
+    --topic-arn arn:aws:sns:us-east-1:000000000000:failed-resize-topic \
+    --protocol email \
+    --notification-endpoint my-email@example.com
+{{< / command >}}
+
+#### Create the Presign Lambda
+
+{{< command >}}
+$ (cd lambdas/presign; rm -f lambda.zip; zip lambda.zip handler.py)
+$ awslocal lambda create-function \
+    --function-name presign \
+    --runtime python3.9 \
+    --timeout 10 \
+    --zip-file fileb://lambdas/presign/lambda.zip \
+    --handler handler.handler \
+    --role arn:aws:iam::000000000000:role/lambda-role \
+    --environment Variables="{STAGE=local}"
+$ awslocal lambda wait function-active-v2 --function-name presign
+$ awslocal lambda create-function-url-config \
+    --function-name presign \
+    --auth-type NONE
+{{< / command >}}
+
+#### Create the Image List Lambda
+
+{{< command >}}
+$ (cd lambdas/list; rm -f lambda.zip; zip lambda.zip handler.py)
+$ awslocal lambda create-function \
+    --function-name list \
+    --handler handler.handler \
+    --zip-file fileb://lambdas/list/lambda.zip \
+    --runtime python3.9 \
+    --timeout 10 \
+    --role arn:aws:iam::000000000000:role/lambda-role \
+    --environment Variables="{STAGE=local}"
+$ awslocal lambda wait function-active-v2 --function-name list
+$ awslocal lambda create-function-url-config \
+    --function-name list \
+    --auth-type NONE
+{{< / command >}}
+
+#### Build the Image Resizer Lambda
+
+{{< tabpane >}}
+{{< tab header="macOS" lang="shell" >}}
+cd lambdas/resize
+rm -rf libs lambda.zip
+docker run --platform linux/x86_64 -v "$PWD":/var/task "public.ecr.aws/sam/build-python3.9" /bin/sh -c "pip install -r requirements.txt -t libs; exit"
+cd libs && zip -r ../lambda.zip . && cd ..
+zip lambda.zip handler.py
+rm -rf libs
+cd ../..
+{{< /tab >}}
+{{< tab header="Linux/Windows" lang="shell" >}}
+cd lambdas/resize
+rm -rf package lambda.zip
+mkdir package
+pip install -r requirements.txt -t package
```

Comment:
```suggestion
pip install -r requirements.txt -t package --platform manylinux_2_28_x86_64 --python-version 3.9 --no-deps
```

This needs to be changed in the requirements.txt or in the command. As you are utilizing Linux lambda to execute it, it is necessary that you specify the platform and version to align with it. If you do not, pip will download Windows version of the python package.